gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs/api-runtime-logs.component.ts

```diff
@@ -47,6 +48,19 @@ export class ApiRuntimeLogsComponent implements OnInit, OnDestroy {
       map((plans) => plans.data),
       shareReplay(1),
     );
+  entrypoints$ = forkJoin([this.api$, this.connectorPluginsService.listEntrypointPlugins()]).pipe(
+    map(([api, plugins]) => {
+      const listeners =
+        api.definitionVersion === 'V4'
+          ? api.listeners.flatMap((listener) => listener.entrypoints).flatMap((entrypoint) => entrypoint.type)
+          : [];
+      return plugins
+        .map((plugin) => {
+          return { id: plugin.id, name: plugin.name };
+        })
+        .filter((plugin) => listeners.includes(plugin.id));
```

Comment:
why applying this filtering ?
let's say you remove an entrypoint from your API, with that filtering, you are not able to filter on old logs that used an other entrypoint.
I am not saying it is wrong, but it can be weird. 
what do you think ?

Reply:
That's a very good point. Also, it will simplify the code a lot. Thanks